### PR TITLE
add mentor others tab, compiler working groups

### PR DESCRIPTION
**Pull request by nikomatsakis**
I added a "Mentor Others" tab as well as setting up the compiler working groups. Here is a brief write-up:

https://gist.github.com/nikomatsakis/1aef48e9bba4a057f5791fbca20d72a6

### CODE REVIEW

File: data/categories.json [hunk 1]
```diff
@@ -60,5 +60,68 @@
...
     "links": [],
     "tags": ["compiler", "lang-rust"]
+},
+{
+    "id": "compiler-errors",
+    "title": "compiler (error messages)",
+    "description": "The Rust compiler",
+    "repository": "rust-lang/rust",
+    "labels": ["WG-compiler-errors"],
+    "links": [ "about" : "https://paper.dropbox.com/doc/Compiler-errors-FSZdfXAGo3uMQ1wuDcZcy" ],
```
Comment (carols10cents) [inline data/categories.json]: These links don't follow [the schemaaaa](https://github.com/nrc/find-work/blob/master/data/schema.md#link)

**Review by carols10cents — CHANGES_REQUESTED** [review]
schema schema bo bema

File: data/categories.json (hunk 1 above)
Comment (nrc) [reply data/categories.json]: ping @nikomatsakis need to fix this up  -should be `[{"text": "about", "url": "..."}]`

**Comment by nikomatsakis** [comment]
@nrc @carols10cents ok I *think* I got it right now ... is there an easy way for me to test?

**Comment by nrc** [comment]
@nikomatsakis you can test findwork locally, but it requires setting up GH auth, etc. I should really have a staging env for this kind of testing

**Comment by nrc** [comment]
At least, findwork does not fail on a bad PR - it only doesn't update

**Pull request merged**